src/dispatchers.ts

```diff
@@ -859,22 +861,75 @@ export function followingFirstCursor() {
     return '0';
 }
 
-export async function outboxDispatcher(
-    ctx: RequestContext<ContextData>,
-    handle: string,
-    cursor: string | null,
+export function createOutboxDispatcher(
+    accountService: AccountService,
+    postService: PostService,
+    siteService: SiteService,
 ) {
-    return {
-        items: [],
-        nextCursor: null,
+    return async function outboxDispatcher(
+        ctx: RequestContext<ContextData>,
+        handle: string,
+        cursor: string | null,
+    ) {
+        ctx.data.logger.info('Outbox Dispatcher');
+
+        const pageSize = Number.parseInt(
+            process.env.ACTIVITYPUB_COLLECTION_PAGE_SIZE || '20',
+        );
+
+        const host = ctx.request.headers.get('host')!;
+        const site = await siteService.getSiteByHost(host);
+        if (!site) {
+            throw new Error(`Site not found for host: ${host}`);
+        }
+
+        const siteDefaultAccount =
+            await accountService.getDefaultAccountForSite(site);
+
+        const outbox = await postService.getOutboxForAccount(
+            siteDefaultAccount.id,
+            cursor,
+            pageSize,
+        );
+        const outboxItems = await Promise.all(
+            outbox.map(async (post: Post) => {
+                return (await buildCreateActivityAndObjectFromPost(post, ctx))
+                    .createActivity;
```

Comment:
Minor, but this gave me a small wtf moment - I thought that we forgot to call `createActivity()`! 

Maybe something like
```ts
const {createActivity} = await buildCreat...
return createActivity;
```

but really up to you!

Reply:
Done!!